src/main/java/org/zalando/nakadi/domain/Timeline.java

```diff
@@ -142,6 +142,10 @@ public void setCleanupAt(@Nullable final Date cleanupAt) {
         this.cleanupAt = cleanupAt;
     }
 
+    public boolean isFake() {
```

Comment:
The timeline which is not registered in DB and equals to the topic of the currently used storage.